packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -507,6 +543,26 @@ export class Database {
     await _indexContent(this, nonCollectionPaths)
   }
 
+  public delete = async (filepath: string) => {
+    const tinaSchema = await this.getSchema()
+    const collection = tinaSchema.schema.collections.find((collection) =>
+      filepath.startsWith(collection.path)
+    )
+    let collectionIndexDefinitions
+    if (collection) {
+      const indexDefinitions = await this.getIndexDefinitions()
+      collectionIndexDefinitions = indexDefinitions?.[collection.name]
+    }
+    await this.store.delete(filepath, {
+      collection: collection.name,
+      indexDefinitions: collectionIndexDefinitions,
+    })
+
+    if (this.store.supportsSeeding()) {
+      await this.bridge.delete(filepath)
+    }
```

Comment:
I'm not sure if this supportsSeeding check is needed? @jeffsee55 thoughts?

Reply:
Yeah I don't think we want this check. @logan-anderson does this all work for users who aren't using the data layer?

Reply:
Yes it does. I will remove it.